### ISSUE TITLE
Bayesian Decision Analysis (Posterior Predictive + Utility Function)

### ISSUE THREAD

**Issue by LarsKarbach** [pymc-devs/pymc-examples#407]
# Notebook proposal
**Title**: Bayesian Decision Analysis

## Why should this notebook be added to pymc-examples?
PyMC lacking an example of a Bayesian Decision Analysis, much in the style of an example in Stan's Users Guide (see reference). I could try to create a Notebook, if that's of interest.

# References
https://mc-stan.org/docs/stan-users-guide/example-decision-analysis.html

**Comment by canyon289** [comment]
this would be great! Please go ahead and add one

**Comment by drbenvincent** [comment]
Seconded, having an example on this topic would be fantastic. Is this still on your radar @LarsKarbach?

**Comment by LarsKarbach** [comment]
Yes, altough i let it slip a bit. It's time to change this! How about give me time until the end of November and see from there if i need help?

**Comment by OriolAbril** [comment]
Took a stab at this in #477, still need to flesh out the text but reviews will be very welcome at any stage.

**Comment by LarsKarbach** [comment]
Hi, very cool. I currently work on an example, which is not running yet. I will continue with this. If we choose to not go with mine, its not to bad.
